### PR TITLE
fix(docs): corrected AuthorizationOptions example line endings

### PR DESCRIPTION
**Pull request by mazizulak**
…ns and put comma

- [x] `npm test` passes on your machine
- [x] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] New tests added or existing tests modified to cover all changes
- [ ] API Documentation in code was updated
- [ ] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

It is a minor fix on documentation.

### CODE REVIEW

File: packages/authorization/README.md [hunk 1]
```diff
@@ -98,8 +98,8 @@ application. The authorization component can be configured with options:
 
 ```ts
 const options: AuthorizationOptions = {
-  precedence: AuthorizationDecisions.DENY;
-  defaultDecision: AuthorizationDecisions.DENY;
+  precedence: AuthorizationDecisions.DENY,
+  defaultDecision: AuthorizationDecisions.DENY
```
Comment (dougal83) [inline packages/authorization/README.md]: nitpick: I believe linting rules would add `,` to the end of both lines.  `defaultDecision: AuthorizationDecisions.DENY,`

**Comment by dougal83** [comment]
Thanks for the contribution!  Could you also amend the commit message?  I would suggest `fix(docs): corrected AuthorizationOptions example line endings`.  This change and the above comment should pass the checks.

File: packages/authorization/README.md (hunk 1 above)
Comment (mazizulak) [reply packages/authorization/README.md]: Done thanks

**Comment by raymondfeng** [comment]
@mazizulak Thank you for the patch.

There are a few minor things to fix:

1. Run `npm run lint:fix` to make sure changed files are formatted correctly
2. Amend the commit log messages
3. Squash all commits into one.

For 2 and 3, you can use `git rebase -i` command. See instructions at https://loopback.io/doc/en/lb4/submitting_a_pr.html.

**Comment by raymondfeng** [comment]
@mazizulak I have cleaned up your commits to fix log message and formatting issues.

**Pull request merged**